tidy3d/components/base.py

```diff
@@ -1397,3 +1397,16 @@ def __getattribute__(self, name: str):
 
     _LazyProxy.__name__ = proxy_name
     return _LazyProxy
+
+
+class InterpolatableMixin(Tidy3dBaseModel):
```

Comment:
I don't understand why this has to be a mixin, instead of an extra field in `ModeInterpSpec`

Reply:
also you may need a unit test for reduce_data, not sure it works currently